Sources/JSONAPI/Resource/Relationship.swift

```diff
@@ -222,7 +222,7 @@ extension Optional: JSONAPIIdentifiable, OptionalRelatable, JSONTyped where Wrap
 }
 
 // MARK: Codable
-private enum ResourceLinkageCodingKeys: String, CodingKey {
+enum ResourceLinkageCodingKeys: String, CodingKey {
```

Comment:
This is for `test_ToManyRelationshipWithMetaNoDataNotOmittable` to check the exact thrown error.